src/coreclr/inc/clrconfigvalues.h

```diff
@@ -749,6 +749,14 @@ RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableHWIntrinsic,  W("EnableHWIntrinsic"),  1
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAES,          W("EnableAES"),          1, "Allows AES+ hardware intrinsics to be disabled")
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAVX,          W("EnableAVX"),          1, "Allows AVX+ hardware intrinsics to be disabled")
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAVX2,         W("EnableAVX2"),         1, "Allows AVX2+ hardware intrinsics to be disabled")
+RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAVX512BW,     W("EnableAVX512BW"),     1, "Allows AVX512BW+ hardware intrinsics to be disabled")
+RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAVX512BW_VL,  W("EnableAVX512BW_VL"),  1, "Allows AVX512BW+ and AVX512VL+ hardware intrinsics to be disabled")
```

Comment:
I wonder if we can reword the "description" better. The concern is that
> "Allows AVX512BW+ and AVX512VL+ hardware intrinsics to be disabled"

Is interpreted as equivalent to `EnableAVX512BW=0` + `EnableAVX512VL=0`, when in practice its independent.

Maybe just this would work, for consistency?
> Allows AVX512BW_VL+ hardware intrinsics to be disabled

Reply:
This comment is still pending.

Reply:
Done